src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int height;
+    private int width;
+    private String color = getColor();
+
+    public Rectangle(int height, int width, String color) {
+        this.height = height;
+        this.width = width;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (height * width);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure rectangle, area: " + areaCalculator()
+                + ", height " + height + ", width " + width
+                + ", color: " + color);
```

Comment:
Use the 'name()' method to get the String representation of enum constants. If 'color' is an enum, you should use 'color.name()' instead of directly referencing the 'color' field.